src/html/popup.html

```diff
@@ -1,33 +1,69 @@
-<!DOCTYPE html>
+<!doctype html>
 <html>
   <head></head>
   <body>
     <div class="uk-container">
-      <div class="uk-margin uk-margin-top">
-        <textarea id="js-title" class="uk-input" type="text" placeholder="title"></textarea>
+      <div class="uk-margin-top">
+        <button
+          id="js-save"
+          class="uk-icon-button uk-icon-large uk-align-center uk-margin-small"
+          uk-icon="bookmark"
+        />
```

Comment:
I decided to change "save button" position to top of the popup window.
I found that most of users click only "databases" and "save"